futurex_openedx_extensions/dashboard/serializers.py

```diff
@@ -947,6 +950,67 @@ def validate_tenant_id(self, value: int) -> int:
             raise serializers.ValidationError('Unable to verify tenant access as request object is missing.')
 
         if value not in request.fx_permission_info['view_allowed_tenant_ids_full_access']:
-            raise serializers.ValidationError(f'User does not have have required access for teanant ({value}).')
+            raise serializers.ValidationError(f'User does not have have required access for tenant ({value}).')
 
         return value
+
+
+class TenantAssetSerializer(ModelSerializerOptionalFields):
```

Comment:
we don't have optional fields here. Do we?
```suggestion
class TenantAssetSerializer(serializers.ModelSerializer):
```